### PR TITLE
update modifications for certificate manager

### PR DESCRIPTION
**Pull request by tianhao-srcn**

### CODE REVIEW

**Review by xujiangang-git — COMMENTED** [review]
have checked change

File: lib/certificateManager.js [hunk 1]
```diff
@@ -1,8 +1,85 @@
 const fs = require('fs');
 const { StepController } = require('./inputStepController');
 const importCertificate = require('./importCertificate');
-const createTizenCertificate = require('./createTizenCertificate');
```
Comment (bouygues) [inline lib/certificateManager.js:4]: change logic for tizen certification management,

**Review by bouygues — APPROVED** [review]
change logic for tizen certification management,

**Review by jianhua-git — COMMENTED** [review]
checked

**Review by chunhua-liu — APPROVED** [review]

**Pull request merged**